xla/service/gpu/custom_call_to_fusion_converter.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2024 The OpenXLA Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef XLA_SERVICE_GPU_CUSTOM_CALL_TO_FUSION_CONVERTER_H_
+#define XLA_SERVICE_GPU_CUSTOM_CALL_TO_FUSION_CONVERTER_H_
+
+#include "absl/container/flat_hash_set.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/string_view.h"
+#include "xla/hlo/ir/hlo_module.h"
+#include "xla/service/hlo_pass_interface.h"
+
+namespace xla {
+namespace gpu {
+
+// Converts custom calls with specific backend configs (currently cuDNN) to
+// fusions with the same backend config.
+class CustomCallToFusionConverter : public HloModulePass {
```

Comment:
Can we give this a more specific name? Do you have specific plans to add more custom call types here?

Reply:
Done. Will probably add more later, no specific plan yet.